packages/documentsearch/src/searchview.tsx

```diff
@@ -723,173 +935,3 @@ class SearchOverlay extends React.Component<ISearchOverlayProps> {
 
   protected translator: ITranslator;
 }
-
```

Comment:
`SearchDocumentView` should not be moved to avoid excessive diffs. The changes in `yarn.lock` should be minimised too.